roxie/ccd/ccdfile.cpp

```diff
@@ -2076,7 +2076,7 @@ class CResolvedFile : implements IResolvedFileCreator, implements ISDSSubscripti
         else
             mb.append(false);
     }
-    virtual ITranslatorSet *getTranslators(int projectedFormatCrc, bool skipFileFormatCrcCheck, IOutputMetaData *projected, IOutputMetaData *expected, RecordTranslationMode mode, bool isIndex) const override
+    virtual ITranslatorSet *getTranslators(int projectedFormatCrc, IOutputMetaData *projected, int expectedFormatCrc, IOutputMetaData *expected, bool skipFileFormatCrcCheck, RecordTranslationMode mode, bool isIndex) const override
```

Comment:
A good change.  I was planning on doing something very similar to the equivalent thor functions when I fixed HPCCC-19590.